packages/core/admin/admin/src/content-manager/pages/ListView/components/BulkActionButtons/tests/index.test.js

```diff
@@ -155,3 +187,117 @@ describe('BulkActionsBar', () => {
     expect(onConfirmUnpublishAll).toHaveBeenCalledWith([1, 2]);
   });
 });
+
+const setupConfirmPublish = () => ({
+  ...render(
+    <ConfirmDialogPublishAll
+      isOpen
+      onConfirm={jest.fn()}
+      onToggleDialog={jest.fn()}
+      isConfirmButtonLoading
+    />,
+    {
+      wrapper({ children }) {
+        const client = new QueryClient({
+          defaultOptions: {
+            queries: {
+              retry: false,
+            },
+          },
+        });
+
+        return (
+          <QueryClientProvider client={client}>
+            <ThemeProvider theme={lightTheme}>
+              <IntlProvider locale="en" messages={{}} defaultLocale="en">
+                <Provider store={store}>
+                  <MemoryRouter>
+                    <Table.Root>{children}</Table.Root>
+                  </MemoryRouter>
+                </Provider>
+              </IntlProvider>
+            </ThemeProvider>
+          </QueryClientProvider>
+        );
+      },
+    }
+  ),
+});
+
+describe('ConfirmDialogPublishAll', () => {
+  beforeAll(() => {
+    server.listen();
+  });
+
+  afterAll(() => {
+    server.close();
+  });
+  it('should show a default message if there are not draft relations', async () => {
```

Comment:
could you add a blank line between the different tests?

Reply:
done